Specs/Number/Arabic/NumberModel.json

```diff
@@ -914,28 +930,28 @@
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "ثلاثة وعشرون وثلاثة أخماس",
+        "Text": "ثلاثة و عشرون و ثلاثة أخماس",
```

Comment:
Why all these changes in specs? Were the original sentences incorrect?
Are they translations of the English specs? We should strive to keep as much coherency of cases as possible (of course, new language-specific cases are very welcome).
There were similar review comments in previous commits in this PR, but no reply.